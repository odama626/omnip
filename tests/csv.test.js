let csv = require('../csv');

test('parse', () => {
	expect.assertions(1);
	return csv.parse(__dirname+'/data/mock_data.csv').then(data => expect(data).toMatchSnapshot());
})