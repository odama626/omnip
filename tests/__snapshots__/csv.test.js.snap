// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse and serialize csv data 1`] = `
Array [
  Object {
    "email": "cherreros0@bigcartel.com",
    "first_name": "Claybourne",
    "gender": "Male",
    "id": "1",
    "ip_address": "14.104.161.197",
    "last_name": "Herreros",
  },
  Object {
    "email": "smclellan1@naver.com",
    "first_name": "Silas",
    "gender": "Male",
    "id": "2",
    "ip_address": "231.230.131.222",
    "last_name": "McLellan",
  },
  Object {
    "email": "trapier2@chronoengine.com",
    "first_name": "Thekla",
    "gender": "Female",
    "id": "3",
    "ip_address": "13.171.205.12",
    "last_name": "Rapier",
  },
  Object {
    "email": "hbartolacci3@blogs.com",
    "first_name": "Haydon",
    "gender": "Male",
    "id": "4",
    "ip_address": "",
    "last_name": "Bartolacci",
  },
  Object {
    "email": "bpaine4@tinypic.com",
    "first_name": "Breena",
    "gender": "Female",
    "id": "5",
    "ip_address": "176.230.87.155",
    "last_name": "Paine",
  },
  Object {
    "email": "hchell5@biglobe.ne.jp",
    "first_name": "Hersh",
    "gender": "Male",
    "id": "6",
    "ip_address": "232.141.193.78",
    "last_name": "Chell",
  },
  Object {
    "email": "klexa6@yahoo.co.jp",
    "first_name": "Kimbell",
    "gender": "Male",
    "id": "7",
    "ip_address": "125.194.113.56",
    "last_name": "Lexa",
  },
]
`;
